envs/home-assistant/shell.nix

```diff
@@ -15,7 +15,9 @@ pkgs.mkShell {
   ];
   # fix homeassistant-pyozw build
   NIX_CFLAGS_COMPILE = [ "-Wno-error=format-security" ];
+  LD_LIBRARY_PATH = pkgs.lib.makeLibraryPath [ pkgs.stdenv.cc.cc ];
   buildInputs = with pkgs; [
+    gcc
```

Comment:
Are you show about this one? 
Using `pkgs.mkShell` should already give you a compiler (gcc on Linux).

Reply:
That is true! Removed it.